react/features/chat/middleware.js

```diff
@@ -255,6 +257,20 @@ function _addChatMsgListener(conference, store) {
     conference.on(
         JitsiConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
         (...args) => {
+            let featureFlag = false;
+            const state = store.getState();
+
+            if (navigator.product === 'ReactNative') {
```

Comment:
Can you please encapsulate this check in a function in the reactions feature (`isReactionsEnabled(state)` for example) and use it everywhere we need to know if they are enabled or not?